plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -1522,8 +1510,26 @@ auto spawnOverviewActor(OverviewActor.Impl self, FlowControlActor.Address flowCt
         import dextool.plugin.mutate.backend.database.type : FileScore;
 
         auto fileScores = ctx.state.get.db.getMutationFileScoreHistory();
+
         ctx.state.get.files.toIndex(content, HtmlStyle.fileDir, fileScores);
 
+        //Change column: timeframe and value calculation handling
+        if (fileScores.length > 0) {
+            auto mut_data = appender!(string[])();
+            mut_data.put("var file_score_data = {};");
```

Comment:
i think this variable need to always be created otherwise there will be JS errors when there are no file score but js-code try to use the variable?